### ISSUE TITLE
ERROR: Server not available

### ISSUE THREAD

**Issue by XmemphistoX** [d8ahazard/Phlex#109]
When trying to test my playback server connectivity - I get the following error

ERROR: MediaServer not available.

Checking the logs I get the following

Tue, 18 Jul 2017 08:45:20 +0200 - util.php::curlGet
Unexpected HTTP code: 401, URL: https://192-168-1-117.xxxxxxxxxxxxxxxxxxxxxx.plex.direct:32400/library/sections?X-Plex-Token=xxxxxxxxxxxxxxx

**Comment by Cyberchaos** [comment]
Also having the same problem when it was working perfectly yesterday. It can see the name of my media server, but won't connect, saying it's unavailable.

**Comment by d8ahazard** [comment]
See my other comments - delete "dlist" and anything with "plexServer" in config.ini.php.  Log out and in.

If nojoy, try toggling 'No Plex.Direct" in Settings>Plex.  Rescan from main menu.  

Still nojoy?  Send all the logs to my donate address - listed in the readme.

**Comment by XmemphistoX** [comment]
Thanks!  That did it.  After the 2nd try though.  Not sure what you meant by reading your other comments (No idea where those are at) so apologies if I missed some FAQ or something where this was already covered.

So detailed actions I took -

Try Number 1-  I deleted dlist and everything with plexServer in it.  Restarted and logged back in.  It detected my friends plex server first which I tested first - It was a success, however when I moved on to my local server, it failed.

Try Number 2 - I deleted dlist and everything with plexServer in it.  Restarted and logged back in.  I waited for it to populate my local server and only tested using that one.  This time it was a success and I left it at that.
